### DISCUSSION TITLE
[Question]: Integration person occupancy detection with no event generated.  What's under the hood?

### DISCUSSION THREAD

**Discussion by bluezclouds** [blakeblackshear/frigate#12315] [Ask A Question] [answered]
### What is your question:

I've been running Frigate for about 3 months now and once Object Detection was properly set up there (amazingly) wasn't a single false alarm.  I set up an automation to send an alarm to my phone if Frigate triggered a occupancy alert through the integration and it's been perfect.  Until this week.

A 4 AM Thursday and again at 4:15 AM last night I was alerted to an intruder inside my fence.  There was no intruder, and to my surprise there was no event generated either time.  There was a short recording with some rain and a cobweb.  (Min and Max filters added to the config should stop that.) 

My questions:  What is the difference between occupancy detection that doesn't trigger an event recording and detection triggering event recording?  Does the object detection need to last a certain amount of time before a event recording is created?

Thanks

**Comment by NickM-27** [comment]
Objects for events have to have moved. Most likely this false positive never moved. Also If you have required zones setup then those zones are not considered for the main sensor.

**Reply by bluezclouds** [reply]
Thanks for the response.

So objects can trigger the person occupancy entity in the integration without moving, but won't create an event.  

Is there any way to tell what actually caused the false positives without that generated event?  If it's something in the yard I'd like to make sure I've included it in my motion and object masks.

**Reply by NickM-27** [reply, marked as answer]
using the /events mqtt topic is recommended for this reason, it includes a lot of information about the object each step of the way

**Reply by bluezclouds** [reply]
Spent most of the day reading up on MQTT and have a test alert set up.  Thanks for pointing me in the right direction.

**Comment by DrSpaldo** [comment]
I’ve also started getting this. The person occupancy is triggering in Home Assistant (Frigate integration) on one of the cameras. But then there is no event or anything to review in Frigate.

Why would it create the person in the HA sensor but not in Frigate itself?

**Reply by NickM-27** [reply]
Probably because the object detected never moved. Events / review items require movement from the position the object was first detected in but the sensor does not.

**Reply by DrSpaldo** [reply]
Interesting. But when I then go to debug on that camera, there is no person …

**Reply by NickM-27** [reply]
While the sensor still says there is?

**Reply by DrSpaldo** [reply]
> While the sensor still says there is?

I'll have to check again when it happens. So would it be in the mqtt sensor? This one? frigate/events

**Reply by NickM-27** [reply]
Yes, that topic is updated even for objects that don't have an event created.
